Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/adapters/GameAdapter.java

```diff
@@ -83,6 +85,22 @@ public void onBindViewHolder(GameViewHolder holder, int position)
     }
 
     holder.gameFile = gameFile;
+
+    Animation animateIn = AnimationUtils.loadAnimation(context, R.anim.anim_card_game_in);
+    animateIn.setFillAfter(true);
+    Animation animateOut = AnimationUtils.loadAnimation(context, R.anim.anim_card_game_out);
+    animateOut.setFillAfter(true);
+    holder.binding.getRoot().setOnFocusChangeListener((v, hasFocus) ->
+    {
+      if (hasFocus)
+      {
+        holder.binding.cardGameArt.startAnimation(animateIn);
+      }
+      else
+      {
+        holder.binding.cardGameArt.startAnimation(animateOut);
+      }
```

Comment:
If you want to make it look a little neater:

```suggestion
      holder.binding.cardGameArt.startAnimation(hasFocus ? animateIn : animateOut);
```

Reply:
I like that